src/patternfly/components/Dropdown/examples/dropdownKebab.hbs

```diff
@@ -0,0 +1 @@
+{{#> dropdown id="dropdown-example-kebab" expanded="true" styleModifier="pf-is-ghost pf-sans-arrow"}}[Kebab]{{/dropdown}}
```

Comment:
what happened with the 💩 ?

Reply:
I was afraid I might step in it. 
![bitmoji](https://render.bitstrips.com/v2/cpanel/ca46077b-19a4-41d8-95d2-0bc33146e21c-fe7ba5ad-9c01-4dac-b094-998c3f204a3b-v1.png?transparent=1&palette=1&width=123)